src/settings.c

```diff
@@ -372,6 +372,16 @@ static void on_settings(struct golioth_client *client,
                         size_t payload_size,
                         void *arg)
 {
+    if (GOLIOTH_OK != response->status)
+    {
+        GLTH_LOGE(TAG,
+                  "Settings callback received status error: %d  CoAP error: %d.%02d",
+                  response->status_code,
+                  response->status_class,
+                  response->status_code);
```

Comment:
We're logging `response->status_code` twice, did you mean to log `response->status`?
```suggestion
        GLTH_LOGE(TAG,
                  "Settings callback received status error: %d  CoAP error: %d.%02d",
                  response->status,
                  response->status_class,
                  response->status_code);
```

Reply:
Thanks for catching this!